COCO_DATA_FORMAT.md

```diff
@@ -71,3 +71,19 @@ Here is one example of the json file for image text matching task. `match: 1` in
   ],
 }
 ```
+
+
+## Image matting
+
+Here is one example of the json file for image matting task. 
+
+``` {json}
+{
+    "images": [{"id": 1, "file_name": "train_images.zip@image/04035_input.jpg"},
+                {"id": 2, "file_name": "train_images.zip@image/01226_input.jpg"}],
+    "annotations": [
+        {"id": 1, "image_id": 1, "label": "image_matting_test_data.zip@mask/04035_input.png"},
```

Comment:
need more info about what this could be and an example would be nice as well

Reply:
what "this“ refers to? the parameter "label"?